Gems/OpenParticleSystem/Assets/Materials/OpenParticle/Types/MaterialInputs/DetailMapsPropertyGroup.json

```diff
@@ -0,0 +1,187 @@
+            "displayName": "    Texture",
+            "description": "Detailed Base Color Texture",
+            "type": "Image",
+            "connection": {
+                "type": "ShaderInput",
+                "name": "m_detail_baseColor_texture"
+            }
+        },
+        {
+            "name": "baseColorDetailBlend",
+            "displayName": "    Blend Factor",
+            "description": "How much to blend the detail layer into the base color.",
+            "type": "Float",
+            "defaultValue": 1.0,
+            "min": 0.0,
+            "max": 1.0,
+            "connection": {
+                "type": "ShaderInput",
+                "name": "m_detail_baseColor_factor"
+            }
+        },
+        {
+            "name": "enableNormals",
+            "displayName": "Enable Normal",
+            "description": "Enable detail normal map to be used for fine detail normal such as scratches and small dents",
+            "type": "Bool",
+            "defaultValue": false
+        },
+        {
+            "name": "normalDetailStrength",
+            "displayName": "    Factor",
+            "description": "Strength factor for scaling the Detail Normal",
+            "type": "Float",
+            "defaultValue": 1.0,
+            "min": 0.0,
+            "softMax": 2.0,
+            "connection": {
+                "type": "ShaderInput",
+                "name": "m_detail_normal_factor"
+            }
+        },
+        {
+            "name": "normalDetailMap",
+            "displayName": "    Texture",
+            "description": "Detailed Normal map",
+            "type": "Image",
+            "connection": {
+                "type": "ShaderInput",
+                "name": "m_detail_normal_texture"
+            }
+        },
+        {
+            "name": "normalDetailFlipX",
+            "displayName": "    Flip X Channel",
+            "description": "Flip Detail tangent direction for this normal map.",
+            "type": "Bool",
+            "defaultValue": false,
+            "connection": {
+                "type": "ShaderInput",
+                "name": "m_detail_normal_flipX"
+            }
+        },
+        {
+            "name": "normalDetailFlipY",
+            "displayName": "    Flip Y Channel",
+            "description": "Flip Detail bitangent direction for this normal map.",
+            "type": "Bool",
+            "defaultValue": false,
+            "connection": {
+                "type": "ShaderInput",
+                "name": "m_detail_normal_flipY"
+            }
+        }
+    ],
+    "propertyGroups": [
+        { 
+            "$import": { 
+                "filename": "UvPropertyGroup.json", 
+                "patch": {
+                    "name": "detailUV",
+                    "description": "Properties for modifying detail layer UV.",
+                    // We need to replace the functors with one that connects to m_detailUvMatrix and m_detailUvMatrixInverse
```

Comment:
O3DE's json support does not cover jsonc (json with comments), if you need to add a comment you might need to add a key//value pair:
```
"comment": "We need to replace the functors with one that connects to m_detailUvMatrix and m_detailUvMatrixInverse"